libs/application/template-api-modules/src/lib/modules/templates/accident-notification/attachments/attachment.utils.ts

```diff
@@ -0,0 +1,97 @@
+import { Application, getValueViaPath } from '@island.is/application/core'
+import {
+  ReviewAddAttachmentData,
+  SubmittedApplicationData,
+} from '@island.is/application/templates/accident-notification'
+import { AccidentNotificationAttachmentStatus } from '../types/applicationStatus'
+import {
+  AccidentNotificationAttachment,
+  AccidentNotificationAttachmentGatherRequest,
+} from '../types/attachments'
+import {
+  additionalFilesRequest,
+  allAttachmentRequestConfig,
+  injuryCertificateRequest,
+  policeReportRequest,
+  powerOfAttorneyRequest,
+  additionalFilesFromReviewerRequest,
+} from '../config'
+
+export const attachmentStatusToAttachmentRequests = (
+  receivedAttachments?: AccidentNotificationAttachmentStatus,
+): AccidentNotificationAttachmentGatherRequest[] => {
+  if (!receivedAttachments) return allAttachmentRequestConfig.requests
+
+  const attachmentRequests: AccidentNotificationAttachmentGatherRequest[] = []
+
+  if (
+    !receivedAttachments.InjuryCertificate &&
+    receivedAttachments.InjuryCertificate != null
```

Comment:
Couldn't you just do this? I feel like it is more readable
```suggestion
     receivedAttachments?.InjuryCertificate === false
```

Reply:
Well the receivedAttachement object is not null it is the InjuryCertificate that the null check is for.

Reply:
Tests fail if i change the implementation to this suggestion